src/browser/PasskeyUtils.cpp

```diff
@@ -319,7 +318,10 @@ QByteArray PasskeyUtils::buildExtensionData(QJsonObject& extensionObject) const
 
     auto extensionData = m_browserCbor.cborEncodeExtensionData(extensionObject);
     if (!extensionData.isEmpty()) {
-        return extensionData;
+        ExtensionResult result;
+        result.extensionData = extensionData;
+        result.extensionObject = m_browserCbor.getJsonFromCborData(extensionData);
+        return result;
```

Comment:
This just reads like we are cbor encoding json data, then decoding it right back to json again. Is that really true? Why do we go back and forth when we already know what the json is?

Reply:
Actually we don't. Result of `{"credProps":true,"uvm":true}` should be `{credProps: {rk: true},uvm: [[1, 1, 1]]}`. We should create the JSON first and the CBOR dynamically for it. I'll make the fixes.

Reply:
I did not do any dynamic handling for the JSON -> CBOR. I think it's too overkill for now with only two extension supported.